.github/workflows/build_android.yml

```diff
@@ -27,10 +27,11 @@ jobs:
               env:
                 DATA: ${{ secrets.GOOGLESERVICES }}
               run: echo $DATA > /home/runner/work/pola-flutter/pola-flutter/android/app/google-services.json
+            # - name: Check formatting
+            #   run: flutter format --set-exit-if-changed #not working in my computer
+            - name: Statically analyze  
```

Comment:
Proponuję dodać odpowiednie zmiany w `.github/workflows/tests.yml`.
Workflow build_android/ios zostawiłbym tylko na sprawdzenie czy apka buduje się na daną platformę. Wszystkie lintery, testy czy formattery które sprawdzają kod flutterowy do workflow tests. Można zmienić mu nazwę by było to bardziej czytelne.